### DISCUSSION TITLE
Using tbot running in docker container

### DISCUSSION THREAD

**Discussion by jux1e** [gravitational/teleport#39948] [Q&A]
Right now im running tbot-distroless in docker container. I've binded /opt/machine-id in container to directory on host.

How do I use ssh_config generated by tbot in that directory with ansible running outside this container (on host)?

**Comment by webvictim** [comment]
If the host's `/opt/machine-id` directory is mounted inside the container, `tbot` should output files to it which Ansible can then use. Do you see files inside the directory after the container runs?
